### DISCUSSION TITLE
ItemAction уведомления + подтверждение

### DISCUSSION THREAD

**Discussion by IgorPetrovPRO** [moonshine-software/moonshine#350] [Ideas]
ItemAction не важно как выполнился коллбэк (успешно или нет) всегда выдает одно и тоже сообщение, кажется полезным, если будет возможность указать разные сообщения для успешного и ошибочного исполнения функции.

Когда добавляем ->withConfirm() всегда показывается одинаковое подтверждение, для кастомизации хотелось бы еще иметь возможность изменить заголовок, текст, текст на кнопке + возможно передать параметры модели.

**Comment by lee-to** [comment]
Решение уже готово https://github.com/moonshine-software/moonshine/pull/352

Появится в новом релизе в ближайшее время

**Reply by IgorPetrovPRO** [reply]
Супер, спасибо, очень жду релиза.
